src/app/dialogs/edit-poll-item-dialog/edit-poll-item-dialog.component.html

```diff
@@ -0,0 +1,117 @@
+<!--
+  ~ Copyright © Live-Poll 2020-2021. All rights reserved
+  -->
+
+<nz-modal [(nzVisible)]="isVisible" (nzOnCancel)="handleCancel()" nzWidth="800px" [nzTitle]="dialogTitle"
+          [nzContent]="dialogContent" [nzFooter]="dialogFooter" (close)="handleCancel()">
+  <!-- Header -->
+  <ng-template #dialogTitle>
+    Edit Poll Item - {{step > stepLabels.length ? 'Updating ...' : stepLabels[step -1]}}
+  </ng-template>
+
+  <!-- Content -->
+  <ng-template #dialogContent>
+    <nz-layout class="dialog-content">
+      <!-- Side bar with timeline -->
+      <nz-sider class="dialog-sidebar" nzWidth="250px">
+        <nz-steps [nzCurrent]="step -1" nzDirection="vertical" nzSize="small">
+          <nz-step *ngFor="let name of stepLabels" nzTitle="{{name}}"></nz-step>
+          <nz-step *ngIf="step === 4" nzTitle="Creating ..." nzStatus="process" nzIcon="loading"></nz-step>
+        </nz-steps>
+      </nz-sider>
+
+      <!-- Main content -->
+      <nz-content>
+        <!-- Step 1 - Enter question / answers -->
+        <div *ngIf="step === 1">
+          <label>
+            <span class="input-label">Question:</span>
+            <input nz-input placeholder="Enter your question ..." [(ngModel)]="question" />
+          </label>
+          <nz-divider></nz-divider>
+
+          <!-- Open Text question / Word cloud question / Rating question -->
+          <div *ngIf="pollItem.type == 'open-text'">
```

Comment:
Wouldn't `[ngSwitch]` be a better fit here? See the [Angular documentation](https://angular.io/api/common/NgSwitch).

Reply:
Fixed it